src/components/GlobalPreferences/HelpAndFeedback/HelpAndFeedback.js

```diff
@@ -1,7 +1,7 @@
 import React, { useCallback } from 'react'
 import { Box, Checkbox, Info, GU, textStyle, useTheme } from '@aragon/ui'
-import helpAndFeedbackSvg from './help-and-feedback.svg'
 import { useHelpScout } from '../../HelpScoutBeacon/useHelpScout'
+import helpAndFeedbackPng from '../../../assets/help-and-feedback.png'
```

Comment:
I'd say this one makes more sense locally, just like Pierre's comment, I try to keep assets locally for component's to be self-contained, when one asset is used more than once (only then) I move them to a global point of import.